README.md

```diff
@@ -119,6 +119,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
     <td align="center"><a href="https://github.com/poojarathore30"><img src="https://avatars1.githubusercontent.com/u/53404072?v=4" width="100px;" alt=""/><br /><sub><b>Pooja Rathore</b></sub></a><br /><a href="https://github.com/TesseractCoding/NeoAlgo/commits?author=poojarathore30" title="Code">💻</a></td>
     <td align="center"><a href="https://github.com/Vrindagupta6828"><img src="https://avatars0.githubusercontent.com/u/50476758?v=4" width="100px;" alt=""/><br /><sub><b>Vrinda Gupta</b></sub></a><br /><a href="https://github.com/TesseractCoding/NeoAlgo/commits?author=Vrindagupta6828" title="Code">💻</a></td>
     <td align="center"><a href="https://shobhittiwari.netlify.com/"><img src="https://avatars3.githubusercontent.com/u/46121289?v=4" width="100px;" alt=""/><br /><sub><b>SHOBHIT TIWARI</b></sub></a><br /><a href="https://github.com/TesseractCoding/NeoAlgo/commits?author=shobhu98" title="Code">💻</a></td>
+    <td align="center"><a href="https://github.com/rayvantsahni"><img src="https://avatars3.githubusercontent.com/u/38404580?s=400&v=4" width="100px;" alt=""/><br /><sub><b>Rayvant Sahni</b></sub></a><br /><a href="https://github.com/rayvantsahni/NeoAlgo/commits?author=rayvantsahni" title="Code">💻</a></td>
```

Comment:
@rayvantsahni you don't have to update the contributors' list before the PR gets merged. Also, there is `allcontributors ` bot which will add the person to the contributor's list automatically just tagging the bot.

Reply:
One of my previous PRs have been already merged. I did not know that the bot automatically does that, apologies for that... So, I dont need to do anything in the `allcontributors` section, right?